From 3f8ba890762513edc7eddbbbd4bb26f908540c8d Mon Sep 17 00:00:00 2001
From: Julien Cristau <jcristau@debian.org>
Date: Sun, 11 May 2008 23:17:27 +0200
Subject: [PATCH] kdrive: allow disabling Composite

KdInitOutput() used to enable Composite when it was disabled by default,
but now this hack prevents ``-extension Composite'' from working.
Remove it, as Composite is enabled by default anyway.
(cherry picked from commit 9dfb525f6c91acab5d1a65765a046bf9ee2aa082)
---
 hw/kdrive/src/kdrive.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/hw/kdrive/src/kdrive.c b/hw/kdrive/src/kdrive.c
index 2bb7b53..cd4ea7d 100644
--- a/hw/kdrive/src/kdrive.c
+++ b/hw/kdrive/src/kdrive.c
@@ -1387,12 +1387,6 @@ KdInitOutput (ScreenInfo    *pScreenInfo,
     KdCardInfo	    *card;
     KdScreenInfo    *screen;
 
-#ifdef COMPOSITE
-    /* kind of a hack: we want Composite enabled, but it's disabled per
-     * default. */
-    noCompositeExtension = FALSE;
-#endif
-    
     if (!kdCardInfo)
     {
 	InitCard (0);
-- 
1.5.5.1

